metadata/service/exporter/configurable/exporter.go

```diff
@@ -46,12 +47,18 @@ func NewMetadataServiceExporter(metadataService service.MetadataService) exporte
 }
 
 // Export will export the metadataService
-func (exporter *MetadataServiceExporter) Export() error {
+func (exporter *MetadataServiceExporter) Export(url *common.URL) error {
 	if !exporter.IsExported() {
 		serviceConfig := config.NewServiceConfig(constant.SIMPLE_METADATA_SERVICE_NAME, context.Background())
 		serviceConfig.Protocol = constant.DEFAULT_PROTOCOL
+		if url == nil || url.SubURL == nil {
+			return errors.New("the url or url.SubURL cannot be nil")
+		}
```

Comment:
Pls modify the error description. Such as 'Metadata Server url is nil, pls check your configuration'. It should be friendly infomation.

Reply:
@sanxun0325  fix or not?

Reply:
@zouyx updated